api-catalog-ui/frontend/src/components/DetailPage/DetailPageContainer.jsx

```diff
@@ -28,6 +28,7 @@ const mapStateToProps = (state) => ({
     fetchTilesError: state.tilesReducer.error,
     selectedTile: state.selectedServiceReducer.selectedTile,
     selectedServiceId: state.selectedServiceReducer.selectedService.serviceId,
+    selectedService: state.selectedServiceReducer.selectedService,
```

Comment:
maybe `selectedServiceId` can be taken from `selectedService` now?

Reply:
could be yes, I will check

Reply:
It eventually turned out that it was not used at all, so I've removed it.